yt/frontends/enzo_p/fields.py

```diff
@@ -0,0 +1,64 @@
+"""
+Fields specific to Enzo-P
+
+
+
+"""
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+from yt.fields.field_info_container import \
+    FieldInfoContainer
+
+rho_units = "code_mass / code_length**3"
+vel_units = "code_velocity"
+acc_units = "code_velocity / code_time"
+energy_units = "code_velocity**2"
+
+known_species_names = {
+}
+
+NODAL_FLAGS = {
+    'BxF': [1, 0, 0],
+    'ByF': [0, 1, 0],
+    'BzF': [0, 0, 1],
+    'Ex': [0, 1, 1],
+    'Ey': [1, 0, 1],
+    'Ez': [1, 1, 0],
+    'AvgElec0': [0, 1, 1],
+    'AvgElec1': [1, 0, 1],
+    'AvgElec2': [1, 1, 0],
+}
```

Comment:
I'd be surprised if this was actually useful for Enzo-p, given that it doesn't support MHDCT yet (right?)